.travis.yml

```diff
@@ -68,14 +68,7 @@ compiler:
 
 install:
         - sudo apt-get install linux-headers-`uname -r`
-script:
-        - if [ -z "${DOCKER_NAMESPACE}" ] ; then export DOCKER_NAMESPACE="opendataplane"; fi
-        - if [ -n "${CROSS_ARCH}" ] ; then
-               docker run  -i -t -v `pwd`:/odp
-                 -e CC="${CC}"
-                 -e CONF="${CONF}"
-                 ${DOCKER_NAMESPACE}/travis-odp-lng-ubuntu_16.04 /odp/scripts/ci/build_${CROSS_ARCH}.sh ;
-          else
+        - if [ -z "${CROSS_ARCH}" ] ; then
```

Comment:
`if [ -z "${CROSS_ARCH}" ] && [ ! -f  /netmap/LINUX/netmap.ko ]`

caching is needed to be enabled.